lua/py-requirements/init.lua

```diff
@@ -13,6 +14,7 @@ local M = {}
 local state = {
     config = {
         enable_cmp = true,
+        requirements_files = { 'requirements.txt' },
```

Comment:
Would you mind changing this to something more generic like `file_patterns`. 

I'm thinking more about supporting other kinds of python dependency files like pyproject.toml and want to leave that a bit more open ended.

Though I probably already leaned too far into the `requirements` convention, oh well.

Reply:
Done!